man/module_case_files.Rd

```diff
@@ -127,28 +127,31 @@
 
 \examples{
 # First, we will initialize a test case file for the 'BioCro' library's
-# 'thermal_time_linear' module, which will be saved in the current directory as
+# 'thermal_time_linear' module, which will be saved in a temporary directory as
 # 'BioCro_thermal_time_linear.csv'. Then, we will add a new case to the file.
 # Finally, we will update the file. Note that the call to `update_csv_cases`
 # will not actually modify the file unless it is manually edited beforehand to
 # change an input or output value.
 
-\dontrun{
+td <- tempdir()
 
 initialize_csv(
   'BioCro:thermal_time_linear',
-  '.',
+  td,
   nonstandard_inputs = list(temp = -1),
   overwrite = TRUE
 )
 
+file.show(file.path(td, 'BioCro_thermal_time_linear.csv'))
```

Comment:
I'm thinking it would be better to show the file contents inline, since it's short.  I think
```
writeLines(readLines(file.path(td, 'BioCro_thermal_time_linear.csv')))
```
would work.  The resulting presentation is clearer, I think.

On my system, at least, I was baffled that clicking "Run examples" showed me output like
```
[1] "Added new case to file `/var/folders/l1/_9rb4hdn4fzdg783664d6yt80000gq/T//Rtmpcs7aM0/BioCro_thermal_time_linear.csv`"
```
but didn't seem to show me the contents of that file.  It was only later, when I was dinking around on my system, that I noticed a small window had appeared, with the file contents shown in a tiny font.  This was when I was working in the R Gui app.  RStudio's behavior was baffling as well.  Instead of opening the file to display it, it prompted me to allow a file called "content" to be saved to my Downloads file.

Reply:
The `writeLines/readLines` method looks good. I just used `file.show` because I was following the examples for `write.csv`. I had no idea it would have such strange behavior. On my computer, it just made a window pop up with the contents of the file, which didn't seem too bad at the time.

Reply:
> On my computer, it just made a window pop up with the contents of the file, which didn't seem too bad at the time.

Just out of curiousity, is this a Windows computer, and are you running R through a GUI app or from the command line?

Reply:
I'm using Windows and RGui. The font size used in the popup window was the same as the font size used in the terminal window.

Reply:
> The font size used in the popup window was the same as the font size used in the terminal window.

Probably more disconcerting than the small font size (which could be made bigger after all) was the fact that the window appeared in the background where I didn't notice it.  In any case, it just makes sense to me to display the file contents "inline", just as the graphs, which take up far more screen real-estate than these small files, are presented inline.